clusterloader2/api/cluster.go

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
I would assume this is in `cmd/` because these are just flags relating to binary.

Reply:
I thought `api` is better place, cause the clusterconfig structure is a some kind of contract. It specifies what information (config fields) can be used by tests and  measurements implementation.

Reply:
I'm not sure if you should call that as API. They're just a set of flags (similar to your test flags) that the binary is needing for some particular things. I'm more inclined to put this into cmd/ (which is where other flags are) or if you really want to separate, put it inside some 'config/' dir.

Reply:
Some e.gs of this pattern:

- https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/options/options.go (for scheduler flags)
- https://github.com/kubernetes/kubernetes/blob/master/cmd/controller-manager/app/options/generic.go (for c-m flags)

Reply:
I've moved this file to `pkg/config/`.
I couldn't move this file to `cmd/`, cause ClusterConfig is used test execution, which is already referenced by `cmd/`. It would cause circular dependency, I think.

Reply:
If you move it to an independent dir `cmd/config/` (like I was suggesting above), I don't see why there should be any circular dependency. But `pkg/config/` also is fine with me.